### DISCUSSION TITLE
Program shutting down with no error when doing : PythonEngine.Initialize();

### DISCUSSION THREAD

**Discussion by quentinformatique** [pythonnet/pythonnet#2401] [Q&A]
### Environment

-   Pythonnet version: 3.0.3
-   Python version: 3.8 and 3.10 and 3.12
-   Operating System: windows 11
-   .NET Runtime: 8.0.302

### Details

-   when I'm triying to run a python code through my .NET program, it crashes at the _PythonEngine.Initialize();_ . I get no errors, the prgram just shuts down

```cs
// <inheritdoc />
public bool CheckValidity(string cube)
{
    PythonEngine.Initialize(); // stop append here
    using (Py.GIL())
    {
        dynamic sys = Py.Import("sys");
        sys.path.append(@"..\py-core");
        dynamic service = Py.Import("core");
        bool solve = service.verify(cube);
        PythonEngine.Shutdown();
        return solve;
    }
}
```

**Comment by JJymsWorld** [comment]
I meet the same problem. Do you have solved the problem?

**Reply by quentinformatique** [reply]
no not yet...
